docs/WEB_INTERFACE_GUIDE.md

```
@@ -10,7 +10,7 @@ The MadSpark Web Interface provides a modern, user-friendly way to generate idea
 
 ```bash
 cd web
-docker-compose up
+docker compose up
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)